core/CoreClient.java

```diff
@@ -135,11 +136,11 @@ void reconnect() {
         }
     }
 
-    TransactionRequestBatcher batcher() {
+    RequestTransmitter executor() {
```

Comment:
I love the name `RequestTransmitter`! But should we now rename this:
```suggestion
    RequestTransmitter transmitter() {
```